src/components/common/mini-image/MiniImage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+
+const MiniImage = ({ unit, mainPhoto }) => {
+
+    return (
+        <>
```

Comment:
This isn't necessary because everything is nested inside a single div anyway. You only need this when you need multiple nodes next to each other in the outermost level.